src/cardboard-ui.js

```diff
@@ -137,6 +138,8 @@ CardboardUI.prototype.listen = function(optionsCallback, backCallback) {
  * Builds the UI mesh.
  */
 CardboardUI.prototype.onResize = function() {
+  if (!WebVRConfig.ENABLE_CARDBOARD_UI) return false;
```

Comment:
handled in PR #51 - included this commit but will remove from this PR when it gets merged